tests/dummy/app/pods/item-primitives/template.hbs

```diff
@@ -0,0 +1,155 @@
+<div style="margin: 30px">
```

Comment:
I find it interesting that you've specified the small/large view in components, but aren't using them here - we can follow up on this after the 30 minute meeting